test/Dialect/FIRRTL/expand-whens.mlir

```diff
@@ -630,4 +630,12 @@ firrtl.module @ModuleWithObjectWire(in %in: !firrtl.class<@ClassWithInput(in in:
   firrtl.propassign %0, %in : !firrtl.class<@ClassWithInput(in in: !firrtl.string)>
 }
 
+// Check that no error is emitted for zero width wires
+// CHECK: firrtl.module @ZeroWidthWire() {
+// CHECK:   %0 = firrtl.wire : !firrtl.uint<0>
+// CHECK: }
```

Comment:
Nit: the PR can skip doing any checks here. It's successful if there's no error.